Missing imports for Logger, HashMap and Map in mapbox-gl InstanceManagerImpl

**Describe the bug**  
Missing imports when RNMapboxMapsImpl is set to mapbox-gl

**To Reproduce**  
Set RNMapboxMapsImpl to mapbox-gl and attempt to build.

**Expected behavior**  
Should build without errors.

**Actual behavior** 
Missing symbols.
```
/maps/android/rctmgl/src/main/java-mapboxgl/mapbox/com/mapbox/rctmgl/impl/InstanceManagerImpl.java:26: error: cannot find symbol
  public static Map<String, String> getTileServers() {
                ^
  symbol:   class Map
  location: class InstanceManagerImpl
/maps/android/rctmgl/src/main/java-mapboxgl/mapbox/com/mapbox/rctmgl/impl/InstanceManagerImpl.java:13: error: cannot find symbol
      Logger.w("InstanceManagerImpl", "setWellKnownTileServer: only mapbox is supported");
      ^
  symbol:   variable Logger
  location: class InstanceManagerImpl
/maps/android/rctmgl/src/main/java-mapboxgl/mapbox/com/mapbox/rctmgl/impl/InstanceManagerImpl.java:27: error: cannot find symbol
    HashMap<String, String> result = new HashMap();
    ^
  symbol:   class HashMap
  location: class InstanceManagerImpl
/maps/android/rctmgl/src/main/java-mapboxgl/mapbox/com/mapbox/rctmgl/impl/InstanceManagerImpl.java:27: error: cannot find symbol
    HashMap<String, String> result = new HashMap();
                                         ^
  symbol:   class HashMap
  location: class InstanceManagerImpl
```

**Versions (please complete the following information):**  
 - git main#21a48e912def5d79f91ab864a5298493f09f0a77

--- a/android/rctmgl/src/main/java-mapboxgl/mapbox/com/mapbox/rctmgl/impl/InstanceManagerImpl.java
+++ b/android/rctmgl/src/main/java-mapboxgl/mapbox/com/mapbox/rctmgl/impl/InstanceManagerImpl.java
@@ -2,6 +2,10 @@ package com.mapbox.rctmgl.impl;
 
 import android.content.Context;
 import com.mapbox.mapboxsdk.Mapbox;
+import com.mapbox.mapboxsdk.log.Logger;
+
+import java.util.HashMap;
+import java.util.Map;
 
 public class InstanceManagerImpl {
   public static void getInstance(Context context, String accessToken) {
@@ -23,7 +27,7 @@ public class InstanceManagerImpl {
     return "mapbox-gl";
   }
 
-  public static Map<String,String> getTileServers() {
+  public static Map<String, String> getTileServers() {
     HashMap<String, String> result = new HashMap();
     result.put("Mapbox", "mapbox");
     return result;

